components/openindiana/pkgbuild/Makefile

```diff
@@ -10,22 +10,23 @@
 
 #
 # Copyright 2011, Andrzej Szeszo
+# Copyright 2019, Nona Hansel
 #
 # Note: Depends on xz for build
 
-include ../../make-rules/shared-macros.mk
+include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		pkgbuild
 COMPONENT_VERSION=	1.3.105
-COMPONENT_REVISION=	2
+COMPONENT_REVISION=	3
```

Comment:
Well, pkgbouild is not so oi-specific, I'd move it to developer

Reply:
Did it, thanks.